client/src/app/api/models.ts

```diff
@@ -885,3 +885,11 @@ export interface SourcePlatform {
   coordinates?: Record<string, any>;
   discoverApplicationsState?: TaskState;
 }
+
+export interface JsonSchemaObject {
+  type: string;
+  properties?: { [key: string]: any };
+  required?: string[];
+  title?: string;
+  description?: string;
+}
```

Comment:
There is https://www.npmjs.com/package/@types/json-schema.  It only covers up to draft 7, but may still be useful